Dockerfile

```diff
@@ -1,38 +1,117 @@
 ARG VERSION=unspecified
+ARG PY_VERSION=3.10.0
 
-FROM python:3.10.0-alpine
+FROM python:${PY_VERSION}-slim-bullseye AS compile-stage
 
 ARG VERSION
 
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 # Note: Additional labels are added by the build workflow.
-LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
+LABEL org.opencontainers.image.authors="jeremy.frasier@cisa.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
 
-ARG CISA_UID=421
-ENV CISA_HOME="/home/cisa"
-ENV ECHO_MESSAGE="Hello World from Dockerfile"
+###
+# Install everything we need
+###
+
+###
+# Dependencies
+#
+# Install dependencies are only needed for software installation and
+# will not be included in the final Docker image.
+###
+ENV DEPS \
+    libpq-dev
+ENV INSTALL_DEPS \
+    wget
```

Comment:
@mcdonnnj is running a tight ship.  

I'd be fine with these being unpinned since they are tied to the base image.   Not saying don't do it, but I'd probably let these float.   

You know how I am.  ☮️

Reply:
They are only tied to what `apt update` pulls down when the image is built. A reproducible Docker build needs to have it all pinned x)

Reply:
Please see commit 0bf0e0f.